modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -84,6 +91,46 @@ protected function register_controls() {
 			],
 		] );
 
+		$repeater->add_control(
+			'selected_icon',
+			[
+				'label' => esc_html__( 'Icon', 'elementor' ),
+				'type' => Controls_Manager::ICONS,
+				'fa4compatibility' => 'icon',
+				'skin' => 'inline',
+				'label_block' => false,
+			]
+		);
+
+		$repeater->add_control(
+			'active_icon',
+			[
+				'label' => esc_html__( 'Icon Active', 'elementor' ),
+				'type' => Controls_Manager::ICONS,
+				'fa4compatibility' => 'icon',
+				'skin' => 'inline',
+				'label_block' => false,
+				'condition' => [
+					'selected_icon[value]!' => '',
+				],
+			]
+		);
+
+		$repeater->add_control(
+			'_element_id',
```

Comment:
```suggestion
			'element_id',
```

Reply:
@matipojo, this is standard for CSS ID (in container , column & section )

Reply:
Done